src/DrawTogether.Actors/Local/LocalDrawingSessionActor.cs

```diff
@@ -93,8 +100,8 @@ public LocalSubscriberDied(IActorRef subscriber)
     private readonly ILoggingAdapter _log = Context.GetLogger();
     private readonly IActorRef _drawingSessionActor;
     private readonly DrawingSessionId _drawingSessionId;
-    private readonly HashSet<IActorRef> _clientSessions = new();
-    private readonly IMaterializer _materializer = Context.System.Materializer();
```

Comment:
This was the major fix - most of the stream actors were staying alive even after the stream was terminated due to the materializer being the `ActorSystem`, rather than this actor instance.